enzyme/Enzyme/ActivityAnalysis.cpp

```diff
@@ -1438,8 +1412,9 @@ bool ActivityAnalyzer::isConstantValue(TypeResults const &TR, Value *Val) {
               F->getName() == "__fd_sincos_1") {
             return false;
           }
-          for (auto FuncName : KnownInactiveFunctionsStartingWith) {
-            if (F->getName().startswith(FuncName)) {
+          auto dName = demangle(F->getName().str());
```

Comment:
Please make sure this works on LLVM 7/8

Reply:
LLVM-7 demangle support seems too basic: https://github.com/llvm/llvm-project/blob/00b161b8971bc6d3cb55f13502288b8fe0dbaa42/llvm/include/llvm/Demangle/Demangle.h

So we now have the demangled support for C++ for >= 9 and the unmangled support for f90io, ss5print and (if <=8) for most C++ IO.

Reply:
diff looks like I missed one place

Reply:
fixed